### DISCUSSION TITLE
Refresh token

### DISCUSSION THREAD

**Discussion by mojixcoder** [fastapi-users/fastapi-users#694] [Q&A]
does `fastapi-users` has something like `djangorestframework-simplejwt` to provide a login response like this:  
```
{
  "token": "asdasdasd",
  "refresh: "adsgdgdds"
}
```  
I want a token with 1 week expire time and a refresh token with 1 month expire time.

**Comment by RadySonabu** [comment]
Have you found an answer to this?

**Comment by AxanIqbal** [comment]
I would like to know too if there is

**Comment by sultanalieva-s** [comment]
As far as I know, the implementation of refresh token logic is not provided by fastapi-users at all. You should implement it manually.
